src/include/schema.h

```diff
@@ -98,10 +98,9 @@ struct __wt_table {
  * WT_WITH_LOCK --
  *	Acquire a lock, perform an operation, drop the lock.
  */
-#define	WT_WITH_LOCK(session, ret, lock, flag, op) do {			\
+#define	WT_WITH_LOCK(session, ret, nowait, lock, flag, op) do {		\
```

Comment:
How about we just make this `WT_WITH_LOCK_NOWAIT` and avoid the extra argument and the `else`?